File: swift/Utils/ApplicationUtils.m
#import "ApplicationUtils.h"

@implementation ApplicationUtils

+(NSString *) getAppName {
    return [[[NSBundle mainBundle] infoDictionary] objectForKey:@"CFBundleDisplayName"];
}

+(NSString *) getAppVersion {
    return [[[NSBundle mainBundle] infoDictionary] objectForKey:@"CFBundleShortVersionString"];
}

+(NSString *) getAppBuild {
    return [[[NSBundle mainBundle] infoDictionary] objectForKey:@"CFBundleVersion"];
}

+(CGSize) getScreenSize {
    return [UIScreen mainScreen].bounds.size;
}

+(CGSize) getApplicationSize {
    return [UIScreen mainScreen].applicationFrame.size;
}

+(NSString *) getPublicDate {
    return [[[NSBundle mainBundle] infoDictionary] objectForKey:@"PublicDate"];
}

@end

File: swift/Utils/ApplicationUtils.swift
import UIKit

class ApplicationUtils: NSObject {
 
    
    class func getAppName() -> String {
        return Bundle.main().objectForInfoDictionaryKey("CFBundleDisplayName") as! String
    }
    
    class func getAppVersion() -> String {
        return Bundle.main().objectForInfoDictionaryKey("CFBundleShortVersionString") as! String
    }
    
    class func getAppBuild() -> String {
        return Bundle.main().objectForInfoDictionaryKey("CFBundleVersion") as! String
    }
    
    class func getScreenSize() -> CGSize {
        return UIScreen.main().bounds.size
    }
    
    class func getApplicationSize() -> CGSize {
        return UIScreen.main().applicationFrame.size
    }
    
    class func getPublicDate() -> String {
        let obj: AnyObject? = Bundle.main().objectForInfoDictionaryKey("PublicDate")
        var str = ""
        if (obj != nil) {
            str = obj! as! String
        }
        return str
    }

}
